docs/vaahcms-2/backend/generate-crud.md

```diff
@@ -110,15 +111,15 @@ MODULE_TRAVEL_ENV=develop
 ## Step-9
 
 In this step you need run `npm install` in terminal.
-```
+```php
 F:\xampp8018\htdocs\vaah_abinash\vaahcms\Modules\Travel>
```

Comment:
@wri-abinash-s001 path `F:\xampp8018\htdocs\vaah_abinash\vaahcms` will not be true for other developers